packages/discord.js/src/structures/interfaces/InteractionResponses.js

```diff
@@ -270,6 +270,7 @@ class InteractionResponses {
   /**
    * Collects a single modal submit interaction that passes the filter.
    * The Promise will reject if the time expires.
+   * <info>Use the `showModal` method to show a modal to the user
```

Comment:
```suggestion
   * <info>Use the `showModal` method to show a modal to the user.</info>
 ```
 
 And it doesnt sound good